frame/support/test/tests/construct_runtime_ui/number_of_pallets_exceeds_tuple_size.stderr

```diff
@@ -55,7 +55,9 @@ help: you might have meant to use the associated type
    |
 38 |     type PalletInfo = Self::PalletInfo;
    |                       ~~~~~~~~~~~~~~~~
-help: consider importing this trait
+help: consider importing one of these items
+   |
+1  + use frame_benchmarking::__private::traits::PalletInfo;
```

Comment:
Shouldn't this suggest

```suggestion
help: consider importing one of these items
   |
1  + use frame_support::traits::PalletInfo;
```

Reply:
Interesting, do you know if it is possible to somehow tell the compiler to ignore that option?

Reply:
maybe @sam0x17 knows